app/javascript/components/Driver/Checkpoints.tsx

```diff
@@ -34,8 +34,8 @@ const Checkpoints: React.FC<CheckpointsFormProps> = (props: CheckpointsFormProps
               <TableRow>
```

Comment:
U need to move { data + align } to another file (create arr) from "StyledTableCell", by use example, and use map to add cells.